### ISSUE TITLE
Cannot use functions

### ISSUE THREAD

**Issue by mvisonneau** [trainline/envmgr-cli#2]
```
~$ envmgr get deploy status d99d4f80-eedc-11e6-8c37-736cf51ee5ca
WARNING:envmgr.parser:Couldn't open envmgr config '/etc/envmgr/get.conf': treating as blank!
Traceback (most recent call last):
  File "/usr/local/bin/envmgr", line 11, in <module>
    sys.exit(envmgr())
  File "/usr/local/lib/python2.7/site-packages/envmgr/command.py", line 30, in envmgr
    os.execvpe(args.command[0], args.command, env.parse())
  File "/usr/local/lib/python2.7/os.py", line 355, in execvpe
    _execvpe(file, args, env)
  File "/usr/local/lib/python2.7/os.py", line 382, in _execvpe
    func(fullname, *argrest)
OSError: [Errno 2] No such file or directory
```

```
~$ envmgr publish MyService-0.0.1.zip as MyService 0.0.1 
WARNING:envmgr.parser:Couldn't open envmgr config '/etc/envmgr/publish.conf': treating as blank!
Traceback (most recent call last):
  File "/usr/local/bin/envmgr", line 11, in <module>
    sys.exit(envmgr())
  File "/usr/local/lib/python2.7/site-packages/envmgr/command.py", line 30, in envmgr
    os.execvpe(args.command[0], args.command, env.parse())
  File "/usr/local/lib/python2.7/os.py", line 355, in execvpe
    _execvpe(file, args, env)
  File "/usr/local/lib/python2.7/os.py", line 382, in _execvpe
    func(fullname, *argrest)
OSError: [Errno 2] No such file or directory
```

**Comment by duncanhall** [comment]
Was this related to the use of Python 3.x or are you still experiencing this issue?

**Comment by mvisonneau** [comment]
This was happening on `python2.7` the version 1.0.0. It was solved with 1.2.0 thanks. Closing this issue.
